### ISSUE TITLE
NPI for bubblesort, training issue

### ISSUE THREAD

**Issue by PlanetMoon** [mokemokechicken/keras_npi#6]
We trying to modify the keras_npi so that can solve the bubblesort as the paper mentioned. The address is https://github.com/PlanetMoon/npi_bubblesort. Thanks a lot about this project.
We noticed that in the function train_f_enc, the pre-training process uses "a,b>=10 & a,b<100" and label is y=a+b+carry. The question is that why choose this condition. In the bubblesort, we trying array length less than 3. However, when we trying training the model with arrays' length equals 3, the training process can't pass the ok rate. We don't know how to change the training set. Are there some tricks for training?

Thanks in advance!

**Comment by mokemokechicken** [comment]
Hi @PlanetMoon 

Although I don't remember very much, there is no special tricks.
First we use easy training data and gradually make it harder.

**Comment by PlanetMoon** [comment]
OK. Thanks! We are using more data to training. Maybe you can help running our project when it is convinient for you?
